dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/config/WSBaseConfig.java

```diff
@@ -203,6 +214,52 @@ public ProtoSignature gaenSigner() {
     }
   }
 
+  @Bean
+  public InsertManager insertManagerExposed() {
+    var manager = new InsertManager(gaenDataService(), gaenValidationUtils());
+    manager.addFilter(new AssertBase64(gaenValidationUtils()));
+    manager.addFilter(new EnforceMatchingJWTClaimsForExposed(gaenRequestValidator));
+    manager.addFilter(new RemoveKeysFromFuture());
+    manager.addFilter(new EnforceRetentionPeriod(gaenValidationUtils()));
+    manager.addFilter(new RemoveFakeKeys());
+    manager.addFilter(new EnforceValidRollingPeriod());
+    return manager;
+  }
+
+  @Bean
+  public InsertManager insertManagerExposedNextDay() {
+    var manager = new InsertManager(gaenDataService(), gaenValidationUtils());
+    manager.addFilter(new AssertBase64(gaenValidationUtils()));
+    manager.addFilter(new EnforceMatchingJWTClaimsForExposedNextDay(gaenValidationUtils()));
+    manager.addFilter(new RemoveKeysFromFuture());
+    manager.addFilter(new EnforceRetentionPeriod(gaenValidationUtils()));
+    manager.addFilter(new RemoveFakeKeys());
+    manager.addFilter(new EnforceValidRollingPeriod());
+    return manager;
+  }
+
+  @ConditionalOnProperty(
+      value = "ws.app.gaen.insertmanager.android0rpmodifier",
+      havingValue = "true",
+      matchIfMissing = false)
+  @Bean
+  public OldAndroid0RPModifier oldAndroid0RPModifier(InsertManager manager) {
+    var androidModifier = new OldAndroid0RPModifier();
```

Comment:
Spring-question: how is this linked with the rest of the code? I don't see any other usage of `[Oo]ldAndroid0RPModifier`?

Reply:
Afaik, some of the filters are optional. Only when the property is set, will this thing be loaded.

Reply:
OK - so an old android TEK will be refused by the `ValidRollingPeriod` filter then?

And if this modifier is enabled, will it be run before or after the filters?

Reply:
modifiers will run before filters

Reply:
Thanks - I think I'll have another go at the README - this was not clear to me.